pkg/agent/openflow/client.go

```diff
@@ -837,12 +842,13 @@ func (c *client) UninstallSNATMarkFlows(mark uint32) error {
 	return c.deleteFlows(c.snatFlowCache, cacheKey)
 }
 
-func (c *client) InstallPodSNATFlows(ofPort uint32, snatIP net.IP, snatMark uint32) error {
-	flows := []binding.Flow{c.snatRuleFlow(ofPort, snatIP, snatMark, c.nodeConfig.GatewayConfig.MAC)}
+func (c *client) InstallPodSNATFlows(ofPort uint32, snatIP net.IP, snatMark, meterId uint32) error {
```

Comment:
We talked it with Zhengdong in the Antrea community meeting. I think here it probably means we enforce the bandwidth limit only on the source Nodes? After some thinking, I feel we should at least also enforce the limit on the egress Node, otherwise the bandwidth limit will not be very useful, assuming source Pods selected by an Egress can be on many different Nodes. @tnqn

Reply:
Checked with Zhengdong offline, another reason why it's difficult to enforce on destination side is that, we don't know which Egress the traffic belongs to when they use same Egress IP (This is possible for the case of static Egress). Could DSCP fields be helpful here?

Reply:
Humm.. that is true. DSCP to encode some information might be a valid idea, but too few bits there to encode much information there.
Should we consider Egress IP based policy?

Reply:
How about uses virtual mac as redirected packets source mac,  then match the source mac in Egress Nodes pipeline. Allocate a global SNAT mark for Egress rule in Antrea Controller, then use the global SNAT mark and EgressIP to generate the virtual mac. @tnqn @jianjuns

Reply:
The problem is the approach sounds too tricky.